src/routes/safe/components/Apps/confirmTransactions.tsx

```diff
@@ -56,16 +55,11 @@ const IconText = styled.div`
   }
 `
 const isTxValid = (t): boolean => {
```

Comment:
just a question, why is `t` untyped here? It's pretty straightforward, isn't it?

Reply:
right, pushed!